### ISSUE TITLE
[a11y design review]: Care Summaries and Notes domain

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#55623]
## Auditor(s)

Sarah Koomson

## Design Review
Based on the development of the prototype, some of the following may or may not be applicable. 
- [x] [Color and contrast checks](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/prepare-for-an-accessibility-staging-review#use-of-color)
  - [x] WCAG 2.1 AA
  - [x] WCAG 2.1 AAA
  - [x] [Who Can Use](https://www.whocanuse.com/?b=ecece5&c=000000&f=20&s=)
  - [x] High Contrast Mode
- [x] [Keyboard navigation](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/prepare-for-an-accessibility-staging-review#keyboard-nav)
  - [x] Annotations for Tab order, reflow and reading order (if needed)
- [x] Accessible Name Annotations
  - [x] [Alt attributes for images](https://webaim.org/techniques/alttext/)
  - [x] Text alternative for complex images such as charts graphs maps etc.
- [x] Language (ease of reading)
- [x] HTML Tags
  - [x] [Page Title](https://accessibility.18f.gov/page-titles/)
  - [x] [Heading order](https://accessibility.18f.gov/headings/)
   - [x] Annotations (if needed)
  - [x] Semantic Consideration for paragraphs ordered lists numbering
  - [x] [Tables (heading rows)](https://design.va.gov/components/table)
- [x] Size of clickable areas

### As Time Allows
- [ ] [Components](https://design.va.gov/components/) follow VA design system guidance.
- [ ] [Patterns](https://design.va.gov/patterns/) follow VA design system guidance.

## Prototype Link

- List View [mobile](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/8D3F2769-3C16-4205-AC55-DA729A666485) | [desktop](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/4F5A3D20-E8A8-4CC5-BDF7-EF943E9DE5CB)
  - Details (Progress Notes) View [mobile](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/72707880-4394-43AC-983F-C6EBC416807E) | [desktop](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/1FE2FF41-CDA8-460E-B7FF-AC8252F59D1A)
  - Details (Admission and discharge summaries) View [mobile](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/7225C1DB-EBFA-47FB-89D0-99752EEF2B2A) | [desktop](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/F59F25FC-03BE-4F24-82AE-CB19F706A838)

Minor (defect 4) **Progress Notes and Admission and Discharge Summary** 
Concise issue description
Busy pages, too much text, too many images and too much other content can cause cognitive overload, anxiety and loss of focus. Keeping content down to a small number of important points reduces the clutter, calms the user, and allows for better understanding while aiding memory.
![DC4D02EA-10F0-4197-926A-3E87ED3ECFE1_1_201_a.jpeg](https://images.zenhubusercontent.com/623cd4d6774110eeb19922cb/55f674c9-5333-4a96-a6c9-e295b4a248c8)Concerns raised of Admission and Discharge Summary and Progress Notes being too content heavy for user. Also could be a really long scroll on mobile. Considered pagination at sync 04/10/2023. 

Bandaid solution

* Because MHV cannot change the content of Progress Notes or the Discharge Summary, our Initial thought was to split information over multiple pages instead of showing it all on a single page (Pagination). 
* However, pagination can feel a bit aggressive. Requiring interaction and effectively breaking find-in-page as a result seems harsh when scrolling is all that is needed. Also - trying to stay away from adding interaction friction. Instead, headings and proper spacing will go a long way in this case if edits can be applied. 

![AEFD6C61-2E6D-48E5-907B-15C268A8ED79_4_5005_c.jpeg](https://images.zenhubusercontent.com/623cd4d6774110eeb19922cb/e25d0941-fd30-4ca1-81c0-b1d34ed9e683)

Be sure to include unique aria-labels for "Details" links

## Related Ticket/Epic
[MHV Accessibility Patterns Consolidation for Midpoint #56578](https://app.zenhub.com/workspaces/sitewide-content-accessibility-and-ia-63a1d63232beba0011a7833f/issues/gh/department-of-veterans-affairs/va.gov-team/56578)

**Comment by sara-amanda** [comment]
## CAIA New Related Ticket

### Related Ticket/Epic
[MHV Accessibility Patterns Consolidation for Midpoint #56578](https://app.zenhub.com/workspaces/sitewide-content-accessibility-and-ia-63a1d63232beba0011a7833f/issues/gh/department-of-veterans-affairs/va.gov-team/56578)

**Comment by SarahKay8** [comment]
@coultonbunney-usds Design review complete. Please close once reviewed. (:

**Comment by RLHecht** [comment]
Closing this ticket. Please reopen if there are any questions. We can open a new ticket for accessibility work for staging.
